zomes/blog/code/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#[macro_use]
+extern crate hdk;
+extern crate holochain_wasm_utils;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+
+pub mod hc_lifecycle_capability;
+pub mod main_capability;
+pub mod post_validations;
```

Comment:
Maybe we could write our own convention in an internal document on how we write 'idiomatic' holochain DNA.
Kind of like starting a 'best practice' document for writing dna.
From Nico: Have a 'syntax' that helps people understand the new way of thinking / paradigm without having it look too much 'like the old stuff'.

Reply:
because here the implied pattern is one rust file per validation, entry_type and capability, and post-fixing the filename with `_capability` or `_validations`

Reply:
One related issue here is: how does this "main_capability" export actually end up getting associated the "main" capability as it would be described in the DNA, since the PATH itself (see file/path rename below) has changed to not include such info